test/unit/theory/theory_strings_rewriter_white.h

```diff
@@ -945,9 +953,7 @@ class TheoryStringsRewriterWhite : public CxxTest::TestSuite
     Node empty_x = d_nm->mkNode(kind::EQUAL, x, empty);
     sameNormalForm(p_xx, empty_x);
 
-    // (str.suffix x (str.++ x x "A")) --> false
-    //
-    // (= x "")
+    // (str.suffix x (str.++ x x "A")) ---> false
```

Comment:
The arguments in this comment of this are (and were previously) flipped.

Reply:
Fixed